packages/components/src/components/button/_mixins.scss

```diff
@@ -42,8 +296,9 @@
     position: absolute;
     right: rem(16px);
```

Comment:
@mattrosno please don't think I'm asking you to make this RFC perfect. I don't expect you to correct this in the RFC necessarily. I'm only trying to point out moments where it doesn't align with our needs to help validate whether this is the correct approach. If something is missing I don't want people to think this is the perfect solution when it's still missing a couple of scenarios. I'll see if @oliviaflory can chime in on this one too.

Reply:
Yes, the button with icon should be positioned 14px from the right in order to achieve the centering within `48x48` box. This will apply to all components with icons positioned on the right.

<img width="596" alt="Screen Shot 2019-08-02 at 3 23 38 PM" src="https://user-images.githubusercontent.com/45692622/62393984-dbfc0200-b539-11e9-9490-d1ef09bcc30e.png">